app/models/data_update_script.rb

```diff
@@ -3,15 +3,14 @@ class DataUpdateScript < ApplicationRecord
   NAMESPACE = "DataUpdateScripts".freeze
   STATUSES = { enqueued: 0, working: 1, succeeded: 2, failed: 3 }.freeze
 
-  default_scope { order(file_name: :asc) }
```

Comment:
`default_scope` is mostly never a good idea, we could have probably kept it in this case since we don't use a `WHERE` condition on it, but within the spirit of building best practices and discouraging things I went ahead and fixed this violation.

For more details on why `default_scope` in Rails is not a good idea see https://github.com/rubocop-hq/rubocop-rails/issues/76